<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>329</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>329</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>329</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="plot_data_box_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Data</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Layer</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QgsMapLayerComboBox" name="ecdf_layer"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QgsRasterBandComboBox" name="ecdf_raster_X">
          <property name="currentText">
           <string/>
          </property>
          <item>
           <property name="text">
            <string>All</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QgsFieldComboBox" name="ecdf_vector_X">
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Leave empty for all fields</string>
          </property>
          <property name="allowEmptyFieldName">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="plot_customization_box_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Plot parameters</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label_113">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QgsColorButton" name="ecdf_color">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_114">
        <property name="text">
         <string>Opacity</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QgsOpacityWidget" name="ecdf_opacity">
        <property name="opacity">
         <double>0.900000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_115">
        <property name="text">
         <string>Log scale</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="ecdf_log_scale">
        <item>
         <property name="text">
          <string>False</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>True</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_112">
        <property name="text">
         <string>ECDF stat</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="ecdf_stat">
        <item>
         <property name="text">
          <string>proportion</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>count</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QgsFieldComboBox" name="ecdf_color_field">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="allowEmptyFieldName">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Color field</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsColorButton</class>
   <extends>QToolButton</extends>
   <header>qgscolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsfieldcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsOpacityWidget</class>
   <extends>QWidget</extends>
   <header>qgsopacitywidget.h</header>
  </customwidget>
  <customwidget>
   <class>QgsRasterBandComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsrasterbandcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
